.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the issue is.
+
+**How to reproduce the bug**
+1. Run the command `ansible-playbook -i inv ...`.
+
+**If applicable, content of the files used in previous command**
+For example, `inv` file from previous sample could contain:
+```
+[mygroup]
+10.5.126.31
+```
+
+**Expected result**
+A clear and concise description of what you expected to happen.
+
+**Actual result**
+A clear and concise description of what actually happened. Make sure you include any error messages, since they usually offer a vital to the bug-fixing process.
+
+
+**Component versions**
+ - Ansible Version [2.9.0]
+ - Sensu Go Collection Version [e.g. 0.7.5]
+
```

Comment:
I will add *Sensu Go Backend Version* to that list because that is indeed important. The agent version is mostly irrelevant to us and this is why I skipped it to reduce the clutter.

As for the supported versions of Sensu Go, our collection supports versions that are marked as *supported* in the documentation: https://docs.sensu.io/sensu-go/5.15/getting-started/versions/

It may work on other, older versions, but we do not guarantee anything there.